### PR TITLE
Config with pure python

### PR DESCRIPTION
**Pull request by icemac**
* Lint the code.
* Add support for Python 3.9 and 3.10.

### CODE REVIEW

**Review by dataflake — APPROVED** [review]
That was a big one, thanks for doing that work.

**Comment by icemac** [comment]
Currently PyPy breaks on Windows, the most important line seems to be:

```
c:\Users\runneradmin\AppData\Local\Temp\pip-install-qnl0eh\persistent\persistent\cPersistence.h(19): fatal error C1083: Cannot open include file: 'bytesobject.h': No such file or directory
```

I am trying to use a more specific PyPy2 version.

**Comment by icemac** [comment]
@icemac wrote:
> I am trying to use a more specific PyPy2 version.

Using the latest PyPy2 release did not help. I created https://github.com/zopefoundation/persistent/issues/167 

I am excluding PyPy2 on Windows for now.

File: docs/conf.py [hunk 1]
```diff
@@ -56,7 +56,7 @@
 
 # General information about the project.
 project = 'ZODB'
-copyright = '2009-2020, Zope Foundation'
+copyright = '2009-2021, Zope Foundation'
```
[the remark this answers is not in the document]
Comment (icemac) [reply docs/conf.py:59]: Implemented in bc13ca74bde916876c43b40f10968ba06f2de733.

**Comment by icemac** [comment]
Thank you for reviewing this PR. 😃

**Pull request merged**